### ISSUE TITLE
[Reproduction] Loss gap (~0.1) when reproducing SFT — requesting clarification on optimizer and unreported hyperparameters

### ISSUE THREAD

**Issue by yulangz** [AweAI-Team/ScaleSWE#8]
你好，感谢开源数据集和 checkpoint！

我正在使用官方发布的轨迹数据和 Qwen3-30B-A3B-Instruct 复现 SFT 训练，已对齐论文 Table 5 中的所有超参数，但发现 loss 收敛于 ~0.4，而用官方 checkpoint 以 lr=0 前向计算的 loss 稳定在 ~0.3，存在约 0.1 的差距。

我当前的优化器配置如下：
```yaml
optimizer:
    type: adam
    lr: 1e-5
    weight_decay: 0.05
    beta1: 0.9
    beta2: 0.95
    eps: 1e-5
    warmup_steps_proportion: 0.05
    lr_scheduler_type: cosine
    gradient_clipping: 1.0
```
loss 曲线：
<img width="970" height="588" alt="Image" src="https://github.com/user-attachments/assets/a4f8d49e-1c65-45fb-aa45-e71e3b4b3534" />
(绿色：我的 SFT 训练；紫色：官方 checkpoint lr=0 前向)

lr 曲线：

<img width="968" height="590" alt="Image" src="https://github.com/user-attachments/assets/430e76c1-c203-4f3b-b0df-e06288c40d06" />

想请教一下下面的这些问题，看看是不是有一些报告中没有提及的超参没对齐：
- 优化器类型是 Adam 还是 AdamW？
- weight_decay、beta1、beta2、eps 分别是多少？
- cosine scheduler 的 min_lr 是否为 0？
- 训练精度是 BF16 吗？

感谢！

**Comment by Heisenburger2020** [comment]
非常抱歉，paper里的epoch数目写错了， 应该是10epoch，下一版upload一下修改一下参数，你说的没有问题，更低的loss才会让指令遵循更好一点

**Comment by yulangz** [comment]
> 非常抱歉，paper里的epoch数目写错了， 应该是10epoch，下一版upload一下修改一下参数，你说的没有问题，更低的loss才会让指令遵循更好一点

感谢！另外可以 share 一下训练的 loss curve 吗～

**Comment by Heisenburger2020** [comment]
<img width="894" height="636" alt="Image" src="https://github.com/user-attachments/assets/13ca0582-3854-48b3-8207-4d77b01f4803" />应该是这个
